### DISCUSSION TITLE
Is there a way to sort songs by the folders they're in?

### DISCUSSION THREAD

**Discussion by Orangian** [DeaDBeeF-Player/deadbeef#2520] [Q&A]
Hi. In AIMP on Windows, it has an option to sort songs by the folder they're in. This is useful since many of my songs don't have proper metadata and are simply sorted through folders. What I want is for songs to be sorted like this while ignoring the metadata:

1.Master Folder
2.Patlabor
3.Music Collection
4.Disc 1

So it actually appears like that in DeaDBeeF, allowing me to more easily find my songs. Is there a way to do this currently?

**Comment by Oleksiy-Yakovenko** [comment]
It should be possible using Edit -> Sort -> Custom -> and entering a title formatting script.
Something like `$directory(%path%)` or `$directory_path(%path%)`, depending on your needs.
You can find more information [here](https://wiki.hydrogenaud.io/index.php?title=Foobar2000:Title_Formatting_Reference#.24directory.28path.29)
